src/test/ui/regions/regions-close-object-into-object-5.base.stderr

```diff
@@ -62,7 +62,7 @@ LL |     Box::new(B(&*v)) as Box<dyn X>
    |                ^^^ ...so that the reference type `&dyn A<T>` does not outlive the data it points at
 
 error[E0310]: the parameter type `T` may not live long enough
-  --> $DIR/regions-close-object-into-object-5.rs:17:16
+  --> $DIR/regions-close-object-into-object-5.rs:21:16
    |
 LL | fn f<'a, T, U>(v: Box<A<T> + 'static>) -> Box<X + 'static> {
    |          - help: consider adding an explicit lifetime bound...: `T: 'static`
```

Comment:
The actual span of this suggestion gets lost in NLL (though we at least suggest it)

Reply:
This too.

Reply:
Created https://github.com/rust-lang/rust/issues/96332